scripts/FigS4_celltype-method-umaps.R

```diff
@@ -146,35 +148,57 @@ ggsave(tcell_umap_file, combined_plot, width = 10, height = 10)
 
 # Consensus/inferCNV UMAPs -----------------------------------------------------
 
-# TODO: Add this in here or move to another script
-# Uncomment out section and clean up code for additional panels
-# colors <- c(palette.colors(n = 8, palette = "Dark2"), "gray80")
-# names(colors) <- levels(umap_df$consensus_lumped)
-# 
-# ggplot(umap_df, aes(x = UMAP.1, y = UMAP.2, color = consensus_lumped)) +
-#   geom_point(
-#     alpha = 0.5,
-#     size = 0.3
-#   ) +
-#   scale_color_manual(values = colors) +
-#   # set points for desired cell type
-#   geom_point(size = 0.3, alpha = 0.5) +
-#   labs(
-#     x = "UMAP1",
-#     y = "UMAP2"
-#   ) +
-#   guides(
-#     color = guide_legend(override.aes = list(alpha = 1, size = 1.5))
-#   )
-# 
-# ggplot(umap_df) +
-#   aes(x = UMAP.1, y = UMAP.2, color = infercnv_total_cnv) +
-#   geom_point(alpha = 0.25, size = 0.25) +
-#   scale_color_viridis_c(na.value = "grey70") +
-#   labs(
-#     x = "UMAP1",
-#     y = "UMAP2",
-#     color = "Total CNV"
-#   ) +
-#   theme(legend.position = "bottom")
-
+# lump consensus cell types 
+umap_df <- umap_df |> 
+  dplyr::mutate(
+    consensus_lumped = forcats::fct_lump_n(consensus_celltype_annotation, n = 7, other_level = "All remaining cell types") |> 
+      #stringr::str_wrap(25) |> # wrap the long names
```

Comment:
I gather this turned out not to be needed

```suggestion
```

I don't think labels are too long with the `(n = {n})` added in either, so this all seems fine.